### PR TITLE
My hacks

### PR DESCRIPTION
**Pull request by wizzardx**
Mainly adding in a few extra stubs that I found helpful.

### CODE REVIEW

File: stubs/3.2/traceback.pyi [hunk 1]
```diff
@@ -3,7 +3,8 @@
 import typing
 
 # TODO signatures
+def format_exception_only(etype, value): pass
```
Comment (spkersten) [inline stubs/3.2/traceback.pyi]: Why did you remove this one?

File: stubs/3.2/queue.pyi [hunk 2]
```diff
@@ -6,5 +6,6 @@ from typing import Any
 
 class Queue():
     def get(self, block: bool = True, timeout: float = None) -> Any: pass
+    def put_nowait(self, item: Any) -> None: pass
```
Comment (spkersten) [inline stubs/3.2/queue.pyi]: While this works, I think you could give the class a type parameter and make that the type of `item` and the return type of `get`.

Comment (wizzardx) [inline stubs/3.2/queue.pyi]: Regarding an earlier comment that I can't find now:

"""
In stubs/3.2/traceback.pyi:

> @@ -3,7 +3,8 @@
>  import typing
>
>  # TODO signatures
> +def format_exception_only(etype, value): pass
Why did you remove this one?
"""

I added that in, rather than removing it.

And regarding this comment:

"""
> @@ -6,5 +6,6 @@ from typing import Any
>
>  class Queue():
>      def get(self, block: bool = True, timeout: float = None) -> Any: pass
> +    def put_nowait(self, item: Any) -> None: pass
While this works, I think you could give the class a type parameter and make that the type of item and the return type of get.
"""

This was just a small change to add an extra method that I needed. If I add type parameters, that would be a separate commit/pull request.

And in general - for now - this is my first pull request. I'm making only minor changes - that I needed for a project - and seeing how this pull request goes, before I make more pull requests.

**Comment by JukkaL** [comment]
Thanks for the updates! It's okay to send small incremental fixes to existing stubs, but new stubs should preferably be reasonably complete (but they don't need to have type annotations and can have `Any` types). Luckily, that's often easy to do because of the stub generator that was recently added.

For `ipaddress`, could you try using the built-in stub generator to get a more complete stub, and update the pull request? For example:

```
$ mkdir out
$ python3 -m mypy.stubgen ipaddress
[stub generated as out/ipaddress.pyi]
```

It's a good idea to manually verify that the generated stub looks reasonable (no need to do a full review).

File: stubs/3.2/queue.pyi (hunk 2 above)
Comment (spkersten) [inline stubs/3.2/queue.pyi]: You can't find the earlier comment because I tried silently removing it before anyone would notice my mistake :smile:.

**Comment by JukkaL** [comment]
I'm currently traveling in Europe but I'll look at this again when I'm back home next week. Sorry for the delay!
